plotting/R_scripts/parallel_plot.R

```diff
@@ -0,0 +1,12 @@
+library(tidyverse)
+library("scales")
+
+parallel <- read.csv("parallel_data.csv")
+
+ggplot() +
+  geom_line(data = parallel, mapping = aes(x = threads, y = insertion_rate), size = 0.5) +
+  #scale_x_continuous(labels=comma, trans="log10") +
```

Comment:
is this comment necessary? if it's meant to offer some kind of customization it might be better to set a file-level variable that toggles this on or off